spec/fixtures/modules/sample/plans/success.pp

```diff
@@ -0,0 +1,5 @@
+plan sample::successful_task (
```

Comment:
This has the wrong name and seems like a duplicate of `successful_task` - remove?